From: Jan Engelhardt <jengelh@inai.de>
Date: 2018-12-06 12:28:42.992186963 +0100

Every new symbol set shipped needs a new SONAME.

Normally, a distro would just ship tarballed releases, and so SONAMEs are just
what upstream puts out.

When shipping snapshots however that have a noncongruent symbol set to any
other release, a custom SONAME will be needed to not claim false compatibility
with any other release.
---
 src/libcamera/base/meson.build |    4 ++--
 src/libcamera/meson.build      |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

Index: libcamera-0.0.0+g3076.d79b4120/src/libcamera/base/meson.build
===================================================================
--- libcamera-0.0.0+g3076.d79b4120.orig/src/libcamera/base/meson.build
+++ libcamera-0.0.0+g3076.d79b4120/src/libcamera/base/meson.build
@@ -27,9 +27,9 @@ libcamera_base_deps = [
 # the use of headers which must not be exposed to the libcamera public api.
 libcamera_base_args = [ '-DLIBCAMERA_BASE_PRIVATE' ]
 
-libcamera_base_lib = shared_library('libcamera-base',
+libcamera_base_lib = shared_library('libcamera-base-suse',
                                     [libcamera_base_sources, libcamera_base_headers],
-                                    version : libcamera_version,
+                                    soversion : '7',
                                     name_prefix : '',
                                     install : true,
                                     cpp_args : libcamera_base_args,
Index: libcamera-0.0.0+g3076.d79b4120/src/libcamera/meson.build
===================================================================
--- libcamera-0.0.0+g3076.d79b4120.orig/src/libcamera/meson.build
+++ libcamera-0.0.0+g3076.d79b4120/src/libcamera/meson.build
@@ -126,9 +126,9 @@ libcamera_deps = [
 # runtime if the library is running from an installed location by checking
 # for the presence or abscence of the dynamic tag.
 
-libcamera = shared_library('libcamera',
+libcamera = shared_library('libcamera-suse',
                            libcamera_sources,
-                           version : libcamera_version,
+                           soversion : '7',
                            name_prefix : '',
                            install : true,
                            include_directories : includes,
